harvester/DBInterface.py

```diff
@@ -950,11 +953,13 @@ def write_record(self, record, repo):
                         try:
                             geopoint["lat"] = float(geopoint["lat"])
                             geopoint["lon"] = float(geopoint["lon"])
+                            # Check coordinates are valid numbers
+                            if not (self.check_lat(geopoint.get("lat")) and self.check_long(geopoint.get("lon"))):
+                                continue
                             extras = {"lat": geopoint["lat"], "lon": geopoint["lon"]}
                             geopoint_id = self.get_single_record_id("geopoint", record["record_id"], **extras)
                             if geopoint_id is None:
                                 self.insert_related_record("geopoint", record["record_id"], **extras)
-                                modified_upstream = True
```

Comment:
@pdante-ubc I think we might need this line--otherwise, adding a new point won't get marked as "modified_upstream". Does that seem right?

Reply:
We aren't actually using Geopoints in Geodisy, at this point, therefor maybe we don't need to adjust modified_upstream so we aren't doing useless redownloads? But if we ever start using points then we will need to add this line back in.

Reply:
That makes sense, I'll update with a comment and merge!